allensdk/brain_observatory/behavior/data_objects/running_speed/running_acquisition.py

```diff
@@ -83,79 +65,52 @@ def __init__(
         self._stimulus_timestamps = stimulus_timestamps
 
     @classmethod
-    @cached(cache=LRUCache(maxsize=10), key=from_json_cache_key)
-    def from_json(
-        cls,
-        dict_repr: dict,
-    ) -> "RunningAcquisition":
-        stimulus_file = BehaviorStimulusFile.from_json(dict_repr)
-        stimulus_timestamps = StimulusTimestamps.from_json(dict_repr=dict_repr,
-                                                           monitor_delay=0.0)
+    def from_stimulus_file(
+            cls,
+            behavior_stimulus_file: BehaviorStimulusFile,
+            sync_file: Optional[SyncFile] = None) -> "RunningAcquisition":
```

Comment:
Can you please clarify in the docstring that the sync file is specifically for reading timestamps? IMO it's more specific if timestamps are an argument rather than a sync file